conda_smithy/lint_recipe.py

```diff
@@ -122,6 +122,20 @@ def lintify(meta, recipe_dir=None):
         lints.append('The recipe `license` should not include the word '
                      '"License".')
 
+    # 11: There should be one empty line at the end of the file.
+    if recipe_dir is not None and os.path.exists(meta_fname):
+        with open(meta_fname, 'r') as f:
+            lines = f.read().split('\n')
```

Comment:
So the trick I found at least is `splitlines` with one terminal newline and zero ends up being the same. 😞 Not sure if it is better in that case.

We can keep the the end line characters though. I'm just not sure if that makes it better or not.